.github/workflows/dev-email-worker.yml

```diff
@@ -11,25 +11,77 @@ on:
     branches:
       - main
 
-jobs:
+jobs:  
```

Comment:
remove whitespace - consider installing "Fix Irregular Whitespace" plugin for you vscode.